test/core/amf-tests.ts

```diff
@@ -112,21 +112,21 @@ describe('Amf', () => {
     doc.requires.set('range/with/resolved', <any>{ range: '1.*', resolved: '1.0.0' });
     strict.equal(doc.requires.get('range/with/resolved')!.raw, '1.* 1.0.0');
 
-    strict.equal(doc.settings.tools.get('CC'), 'foo/bar/cl.exe', 'should have a value');
-    strict.equal(doc.settings.tools.get('CXX'), 'bin/baz/cl.exe', 'should have a value');
-    strict.equal(doc.settings.tools.get('Whatever'), 'some/tool/path/foo', 'should have a value');
+    strict.equal(doc.exports.tools.get('CC'), 'foo/bar/cl.exe', 'should have a value');
```

Comment:
This is a schema change, right? Is anyone on the hook to change the default catalog?

Reply:
Yeah. I have an updated copy of the repo locally (and was going to convert it to all JSON).